cmd/compare.go

```diff
@@ -66,3 +82,137 @@ func rebuildCompareCommand() {
 	compareCmd = createCompareCommand()
 	buildCompareCommand()
 }
+
+// FIXME: duplicated in update.go
```

Comment:
Is this broken here? Or just duplicated?

Reply:
Just duplicated. Should be consolidated somehow.